test/blackbox-tests/test-cases/env-var-expansion/correct/dune

```diff
@@ -0,0 +1,18 @@
+(alias
+ (name echo1)
+ (action (echo %{env:DUNE_ENV_VAR=true})))
+
+(alias
+ (name echo2)
+ (action (setenv DUNE_ENV_VAR false
+          (echo %{env:DUNE_ENV_VAR=true}))))
+
+(alias
+ (name enabled)
+ (enabled_if (= true %{env:DUNE_ENV_VAR=true}))
```

Comment:
Note: that's almost the same as `(enabled_if %{env:DUNE_ENV_VAR})`, except that it would raise if the user pass a value other than `true` or `false` in `DUNE_ENV_VAR`.